test/java/com/odysseusinc/arachne/portal/front/ProfileManagerTest.java

```diff
@@ -0,0 +1,66 @@
+package com.odysseusinc.arachne.portal.front;
+
+import static com.odysseusinc.arachne.portal.front.BaseUserTest.logout;
+import static com.odysseusinc.arachne.portal.front.utils.Utils.waitFor;
+
+import com.odysseusinc.arachne.portal.front.utils.ByBuilder;
+import java.io.IOException;
+import org.apache.commons.lang.StringUtils;
+import org.junit.AfterClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class ProfileManagerTest extends BaseTest {
+
+    @AfterClass
+    public static void afterTest() throws IOException {
+
+        shutdown();
+    }
+
+    @Test
+    public void test01UpdateAdminName() throws Exception {
+
+        updateName(ADMIN_LOGIN, ADMIN_PASSWORD, "4");
+    }
+
+    protected static void updateName(String login, String password, String postfix) {
+
+        loginPortal(login, password);
+
+        By profileMenu = ByBuilder.byClassAndText("ac-profile-menu-item__user-name-part", "admin");
+        waitFor(driver, profileMenu);
+        WebElement profileMenuWebElement = driver.findElement(profileMenu);
+        profileMenuWebElement.click();
+
+        By userProfile = ByBuilder.text("User Profile");
+        waitFor(driver, userProfile);
+        WebElement userProfileWebElement = driver.findElement(userProfile);
+        userProfileWebElement.click();
+
+        waitFor(driver, ByBuilder.byClassAndText("ac-profile-view__user-name", "admin admin admin"));
```

Comment:
Still believe that there is better way to find text inside toobar, than stick to concrete and very specific classes